enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1344 @@
+    # This field is modeled on HCP's existing HostedCluster.spec.pausedUntil which uses a string. 
+    disabledUntil: <datetime|bool>str
+    
+    # If set to "true" or a future date (represented as string), the strategy will be
+    # paused (not permissive) unless overridden by disabledUntil. If paused with this 
+    # setting, it overrides the "policy" stanza.
+    # Date must be RFC3339. 
+    # This field is modeled on HCP's existing HostedCluster.spec.pausedUntil which uses a string. 
+    pausedUntil: <datetime|bool>str  
+    
+    # If not overridden with disabledUntil / pausedUntil, a reference to the ChangeManagementPolicy
+    # to determine whether material changes can be initiated.
+    policy:
+      # Namespace is optional. If not specified, the controller assumes the namespace in
+      # which the controller is running.
+      namespace: openshift-machine-config-operator
+      # The name of the ChangeManagementPolicy.
+      name: example-policy
+```
+
+At a given moment, a `changeManagement` stanza indicates to a controller responsible for a resource
+whether changes should be paused (i.e. it is a restrictive window where material changes should not
+be initiated) or unpaused (i.e. it is a permissive window where material changes can be initiated).  
+- `changeManagement.disabledUntil: "<bool|date>"`: When `disabledUntil: "true"` or `disabledUntil: "<future-date>"`, 
+  the controller completely disables change management and all changes are permitted. `disabledUntil` overrides
+  both `pausedUntil` and `policy` when it suggests change management should be disabled. 
+- `changeManagement.pausedUntil: "<bool|date>"`: When `pausedUntil: "true"` or `pausedUntil: "<future-date>"`, 
+  changes must be paused and the controller must stop initiating material changes. `pausedUntil` overrides
+  `policy` when it suggests changes should be paused.
+- `changeManagement.policy`: An optional reference to a `ChangeManagementPolicy` object. If neither `disabledUntil`
+  or `pausedUntil` overrides it, the permissive or restrictive state suggested by the policy object will 
+  inform the controller whether material changes can be initiated.
+
+While fields like `disabledUntil` or `pausedUntil` may seem to add unnecessarily complexity, they provide
+simple to use knobs for SRE and cluster lifecycle administrators to confidently undertake sensitive actions. 
+For example, 
+- If SRE needs to apply emergency corrective action on a cluster with a `MaintenanceSchedule` change
+  management strategy configured, they can simply set `disabledUntil: <tomorrow>` without having to 
+  change object references OR worry about follow-up corrective actions to restore a previous policy.
+- If the cluster lifecycle administrator needs to urgently stop a problematic update, they can set
+  `pausedUntil: true` until issues are fully understood. In a scenario impacting business critical 

```

Comment:
From the names `disabledUntil` and `pausedUntil`, it is not clear what each of them actually disables or pauses. The terms "disable" and "pause" are too similar and insinuate that `disable` is the stronger version of `pause`; however, for this proposal, they would be the exact opposite operation.

With the current naming scheme, I could easily see an admin who wants to set `pausedUntil: true` because they urgently want to stop a problematic update. Instead, they confuse the 2 terms, and set `disabledUntil: true` which effectively does the exact opposite of what the admin wants